source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -1091,19 +1095,15 @@ void ShaderGraph::populateInputColorTransformMap(ColorManagementSystemPtr colorM
     const string& sourceColorSpace = input->getActiveColorSpace();
     if (shaderInput && !sourceColorSpace.empty())
     {
-        // Can skip inputs with connections as they are not legally allowed to have colorspaces specified.
-        if (!shaderInput->getConnection())
```

Comment:
We can't check for input connections here, since we are in the middle of graph traversal and upstream connections have not been processed yet.